inlang/source-code/sdk/src/v2/mocks/multipleMatcher/bundle.ts

```diff
@@ -0,0 +1,188 @@
+import type { MessageBundle } from "../../types.js"
+
+export const multipleMatcherBundle: MessageBundle = {
+	id: "mock-bundle-human-id",
+	alias: {
+		default: "mock-bundle-alias",
+	},
+	messages: [
+		{
+			locale: "en",
+			declarations: [
+				{
+					type: "input",
+					name: "username",
+					value: {
+						type: "expression",
+						arg: {
+							type: "variable",
+							name: "username",
+						},
+					},
+				},
+			],
+			selectors: [
+				{
+					type: "expression",
+					arg: {
+						type: "variable",
+						name: "count: plural",
+					},
+				},
+			],
+			variants: [
+				{
+					match: ["one"],
+					pattern: [
+						{
+							type: "text",
+							value: "Show {count} message.",
+						},
+					],
+				},
+				{
+					match: ["many"],
+					pattern: [
+						{
+							type: "text",
+							value: "Show {count} messages.",
+						},
+					],
+				},
+				{
+					match: ["*"],
+					pattern: [
+						{
+							type: "text",
+							value: "Show {count}",
+						},
+					],
+				},
+			],
+		},
+		{
+			locale: "de",
+			declarations: [
+				{
+					type: "input",
+					name: "username",
+					value: {
+						type: "expression",
+						arg: {
+							type: "variable",
+							name: "username",
+						},
+					},
+				},
+			],
+			selectors: [
+				{
+					type: "expression",
+					arg: {
+						type: "variable",
+						name: "count: plural",
+					},
+				},
+				{
+					type: "expression",
+					arg: {
+						type: "variable",
+						name: "formal: bool",
```

Comment:
bool should be an annotation (function) as well right?
Do you know how this function would be defined ?
I could not find it in https://github.com/unicode-org/message-format-wg/blob/main/spec/registry.md
cc: @martin-lysk @LorisSigrist for clarification of how the "formal" selector is expected to work.

Reply:
This might also affect how the formal selectors are expressed in other parts of the AST.

Reply:
I did it like the plural. Can we make a separate discussion about it, so we can merge this PR.